content/en/docs/troubleshooting/ClusterResourcePlacementEviction.md

```diff
@@ -0,0 +1,260 @@
+> process where the controller is unable to reach the api server.
+
+The first step in troubleshooting is to check the status of the eviction object to understand if the eviction reached
+a terminal state or not.
+
+## Invalid eviction
+
+### Missing/Deleting CRP object
+
+Example status with missing `CRP` object:
+```
+status:
+  conditions:
+  - lastTransitionTime: "2025-04-17T22:16:59Z"
+    message: Failed to find ClusterResourcePlacement targeted by eviction
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionInvalid
+    status: "False"
+    type: Valid
+```
+
+Example status with deleting `CRP` object:
+```
+status:
+  conditions:
+  - lastTransitionTime: "2025-04-21T19:53:42Z"
+    message: Found deleting ClusterResourcePlacement targeted by eviction
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionInvalid
+    status: "False"
+    type: Valid
+```
+
+In both cases the Eviction object reached a terminal state, it's status has `Valid` condition set to `False`. 
+The user should verify if the `ClusterResourcePlacement` object is missing or if it is being deleted and recreate the 
+`ClusterResourcePlacement` object if needed and retry eviction.
+
+### Missing CRB object
+
+Example status with missing `CRB` object:
+```
+status:
+  conditions:
+  - lastTransitionTime: "2025-04-17T22:21:51Z"
+    message: Failed to find scheduler decision for placement in cluster targeted by
+      eviction
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionInvalid
+    status: "False"
+    type: Valid
+```
+
+In this case the Eviction object reached a terminal state, it's status has `Valid` condition set to `False`, because the
+`ClusterResourceBinding` object or Placement for target cluster is not found. The user should verify to see if the 
+`ClusterResourcePlacement` object is propagating resources to the target cluster,
+
+- If yes, the next step is to check if the `ClusterResourceBinding` object is present for the target cluster or why it 
+was not created and try to create an eviction object once `ClusterResourceBinding` is created.
+- If no, the cluster is not picked by the scheduler and hence no need to retry eviction.
+
+### Multiple CRB is present
+
+Example status with multiple `CRB` objects:
+```
+status:
+  conditions:
+  - lastTransitionTime: "2025-04-17T23:48:08Z"
+    message: Found more than one scheduler decision for placement in cluster targeted
+      by eviction
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionInvalid
+    status: "False"
+    type: Valid
+```
+
+In this case the Eviction object reached a terminal state, it's status has `Valid` condition set to `False`, because
+there is more than one `ClusterResourceBinding` object or Placement present for the `ClusterResourcePlacement` object 
+targeting the member cluster. This is a rare scenario, where one `ClusterResourceBinding` is being deleted while a new 
```

Comment:
Hi Arvind! Just a nit: it might be better to re-phrase this a bit, mentioning that this is a rare occurrence and the user might be observing an **in-between state** where bindings are being-recreated due to the cluster being selected again, and **it will normally resolve quickly**.